src/matchspec.rs

```diff
@@ -430,6 +433,16 @@ impl<S: AsRef<str> + PartialOrd + PartialEq<str> + Into<String>> MatchSpec<S> {
     }
 }
 
+impl MatchSpec<String> {
+    pub fn is_match(&self, pc: &PackageCandidate) -> bool {
+        self.is_package_version_match(
+            &pc.name,
+            &pc.version.as_ref().unwrap_or(&String::new()))
+            && self.subdir == pc.subdir
+            && self.build == self.build
```

Comment:
```suggestion
            && self.build == pc.build
```
I think you meant to do pc here